components/engine/container/health_test.go

```diff
@@ -0,0 +1,15 @@
+package container
+
+import "testing"
+
+func TestOpenCloseMonitorChannel(t *testing.T) {
```

Comment:
This test is extremely silly.

Reply:
I expanded on why in https://github.com/moby/moby/pull/35482#issuecomment-344446125.

Reply:
For sure, in writing new code that does not have bugs I would not see value in having this test. But in a PR to fix a bug, this test proves that without your git commit 74f7c76 the test will fail. The test is simple to maintain, not resource intensive, and does not put up false failures.